Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityToolkitConfigurationProfileInspector.cs

```diff
@@ -248,6 +251,32 @@ protected override void OnEnable()
                     },
                     () => {
                         EditorGUILayout.PropertyField(useServiceInspectors);
+
+                        using (var c = new EditorGUI.ChangeCheckScope())
+                        {
+                            EditorGUILayout.PropertyField(renderDepthBuffer);
+                            if (c.changed)
+                            {
+                                if (renderDepthBuffer.boolValue)
+                                {
+                                    CameraCache.Main.gameObject.AddComponent<DepthBufferRenderer>();
+                                }
+                                else
+                                {
+                                    foreach (var dbr in FindObjectsOfType<DepthBufferRenderer>())
+                                    {
+                                        if (Application.isPlaying)
```

Comment:
Is this a place we can use one of our object destroy extensions?

Reply:
ah it's finally been created